charts/linkerd-control-plane/values.yaml

```diff
@@ -280,6 +280,12 @@ proxy:
       # -- The maximum duration for a response stream (i.e. before it will be
       # reinitialized).
       lifetime: "1h"
+  # Configures proxy metrics
+  metrics:
+    # Configures outbound proxy metrics
+    outbound:
```

Comment:
Given that the annotation is `/proxy-enable-hostname-labels`, it's probably preferable to simply call this `enableHostnameLabels: false`